docs/src/routes.tsx

```diff
@@ -33,11 +33,25 @@ import PopupsPrototype from './prototypes/popups'
 import IconViewerPrototype from './prototypes/IconViewer'
 import MenuButtonPrototype from './prototypes/MenuButton'
 import AlertsPrototype from './prototypes/alerts'
+import { Routes as BrowserTestRoutes } from '../../e2e'
+import { match } from 'react-router'
+
+type E2EProps = {
```

Comment:
Can we use `RouteComponentProps` instead?

```suggestion
type E2EProps = RouteComponentProps<{ exampleName: string }>
```